src/ResourceManager/Sql/Commands.Sql/Data Sync/Cmdlet/StartAzureSqlSyncGroupSync.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Sql.DataSync.Model;
+
+namespace Microsoft.Azure.Commands.Sql.DataSync.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to trigger data synchronization of a sync group
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Start, "AzureRmSqlSyncGroupSync", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.None)]
```

Comment:
This should be ConfirmImpact.Medium

Should have an OutputType attribute for the PassThrough scenario